config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  
```

Comment:
Rubocop warns about not having newline at end of file. Either way, the last line shouldn't have the indentation.